src/flashlight.common.ts

```diff
@@ -0,0 +1,24 @@
+import { Observable } from 'tns-core-modules/data/observable';
+
+export abstract class Common extends Observable {
```

Comment:
Why do you extend Observable?

Reply:
I did not personally do this. This is how the new seed did this. So instead of touching it, I thought of letting it stay there